semgrep-core/src/parsing/pfff/php_to_generic.ml

```diff
@@ -435,9 +435,12 @@ and match_ = function
       let e = expr e in
       G.CasesAndBody ([ G.Default tok ], G.ExprStmt (e, G.sc) |> G.s)
 
-and argument e =
-  let e = expr e in
-  G.arg e
+and argument = function
+  | Arg e -> expr e |> G.arg
+  | ArgRef (tok, e) -> G.Ref (tok, expr e) |> G.e |> G.arg
+  | ArgUnpack (tok, e) ->
+      G.OtherExpr (("Unpack", tok), [ G.E (expr e) ]) |> G.e |> G.arg
```

Comment:
Shouldn't this be `Call (IdSpecial Spread, Id foo)`? See [this comment](https://github.com/returntocorp/semgrep/blob/3849f73dccdda2b5ff461fc2bff9cf17572df3b3/semgrep-core/src/core/ast/AST_generic.ml#L838).